commands/post.go

```diff
@@ -125,11 +132,25 @@ func printPost(c client.Client, post *model.Post, usernames map[string]string, s
 		}
 	}
 
+	postTime := time.UnixMilli(post.CreateAt).UTC()
+	createdAt = postTime.Format(PostTimeFormat)
+
 	if showIds {
-		printer.PrintT(fmt.Sprintf("\u001b[31m%s\u001b[0m \u001b[34;1m[%s]\u001b[0m {{.Message}}", post.Id, username), post)
+		printer.PrintT(fmt.Sprintf("\u001b[31m%s\u001b[0m \u001b[0m \u001b[37;1m%s \u001b[34;1m[%s]\u001b[0m {{.Message}}", post.Id, createdAt, username), post)
 	} else {
-		printer.PrintT(fmt.Sprintf("\u001b[34;1m[%s]\u001b[0m {{.Message}}", username), post)
+		printer.PrintT(fmt.Sprintf("\u001b[0m \u001b[37;1m%s \u001b[34;1m[%s]\u001b[0m  {{.Message}}", createdAt, username), post)
+	}
+}
+
+func getPostList(client client.Client, channelID, since string, perPage int) (*model.PostList, *model.Response, error) {
+	if len(since) > 0 {
+		sinceTime, err := time.Parse(ISO8601Layout, since)
+		if err != nil {
+			return nil, nil, fmt.Errorf("invalid since time format: %v", err)
+		}
+		return client.GetPostsSince(channelID, sinceTime.UTC().UnixMilli(), false)
 	}
+	return client.GetPostsForChannel(channelID, 0, perPage, "", false)
```

Comment:
It would be good to add tests for listing posts. It does look like the existing post listing functionality isn't tested, although there are some examples of other post operations being tested I think we can build off.

Reply:
Agreed, let's add some unit tests and end-to-end tests as well.

Reply:
Added unit and e2e tests